pennylane/ops/op_math/controlled.py

```diff
@@ -82,7 +83,39 @@ def circuit(x):
     >>> qml.simplify(qml.adjoint(op))
     Controlled(RY(12.466370614359173, wires=[0]) @ RX(10.166370614359172, wires=[0]), control_wires=[1])
 
+    **Example**
```

Comment:
Would you update the docstring of `adjoint` by following https://github.com/PennyLaneAI/pennylane/pull/4709?

Reply:
Updated let me know if you have more comments

Reply:
Thanks!